code/traffic-register/traffic-register.css

```diff
@@ -539,6 +539,17 @@ a.ang-link:link {text-decoration: none;}
 /*Draft Reg Doc ID on Draft Review*/
 #kn-scene_685 .kn-detail-label { background-color: transparent; min-width: 0% !important; } /*Edit Area & Remarks*/
 
+/**********************************/
```

Comment:
Is there any way to get a visual or gif on this particular enhancement?

Reply:
you can view the before and after in the original issue: [#20666](https://github.com/cityofaustin/atd-data-tech/issues/20666)